suggestions/tts.json

```diff
@@ -1,8 +1,16 @@
 [
     {
-        "id": "com.danefinlay.ttsutil",
-        "label": "TTS Util",
-        "source": "fg",
-        "repo": "https://github.com/Danesprite/tts-util-app"
+        "id": "com.github.olga_yakovleva.rhvoice.android",
+        "label": "RHVoice",
+        "reason": "No same/better alternative is available right now",
```

Comment:
These reasons state effect rather than the actual cause (e.g. reason). RHVoice only support a few languages including English, Slavic and languages from a few other post-USSR countries but with better quality. eSpeak supports many languages but most of them are incomplete and the quality is not very good.

Reply:
In README.md you said "Interface or functions (UI/UX) should be very or nearly similar to the original app or even better. If no such alternative is available right now, the `reason` must be included."

Reply:
> In README.md you said "Interface or functions (UI/UX) should be very or nearly similar to the original app or even better. If no such alternative is available right now, the `reason` must be included."

Most original TTS apps have the same or similar issues. In fact, I think voice quality of RHVoice is better than some of those bloatware. Google's AI-based engine is only available in a few places such as Google Play Books.